src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3367,6 +3374,27 @@ export class ResultsViewPageStore extends AnalysisStore
         },
     });
 
+    readonly structuralVariantsByGene = remoteData({
+        await: () => [this.structuralVariants],
+        invoke: async () => {
+            const svByGene: Record<string, StructuralVariant[]> = {};
+            this.structuralVariants.result!.forEach(sv => {
+                if (sv.site1HugoSymbol?.length) {
```

Comment:
it seems strange to test the length of this string.    just testing for 
```
if (sv.site1HugoSymbol) { ... }
``` 
should be sufficient

Reply:
Haha it's legacy code. I'll fix it.